float32 batteryVol
float32 thruster_0
float32 thruster_1
float32 thruster_2
float32 thruster_3
float32 fins
float32 prop
